qa/acceptance-tests/src/test/java/io/camunda/it/client/MessageSubscriptionSearchTest.java

```diff
@@ -46,6 +48,15 @@ static void beforeAll() {
 
     waitForMessageSubscriptions(camundaClient, NUMBER_OF_MESSAGE_SUBSCRIPTIONS);
 
+    camundaClient
+        .newCorrelateMessageCommand()
+        .messageName("Message1")
+        .correlationKey("correlation_key_1")
+        .send()
+        .join();
+
+    waitForCorrelatedMessages(camundaClient, 1);
```

Comment:
We should wait for one correlated subscription here via subscription search to avoid flakiness.